experimental/dds/tree2/package.json

```diff
@@ -25,6 +25,7 @@
 		"build:test": "tsc --project ./src/test/tsconfig.json",
 		"ci:build:docs": "api-extractor run --typescript-compiler-folder ../../../node_modules/typescript && copyfiles -u 1 ./_api-extractor-temp/* ../../../_api-extractor-temp/",
 		"clean": "rimraf --glob 'dist' 'lib' '*.tsbuildinfo' '*.build.log' '_api-extractor-temp' 'nyc'",
+		"depcruise": "depcruise src/ --ignore-known",
```

Comment:
I'm guessing that the known issues file is intended to be updated via some automated process? If so, it would be nice to add a script in here that can be invoked to regenerate it.

Reply:
But it would defeat the porpouse of not allowing new issues to be introduced.

Reply:
We want to avoid making general development and refactoring (including ones to fix cycles) harder (ex: if I rename a folder thats part of several cycles, I don't want to have to go update a 100+ lines of stuff in that file). The goal is to help developers understand and better evaluate the implications of their actions. If I add or modify a cycle, I will have to acknowledge that (by regenerating this file), and run that change through code review in the PR. That is sufficient to accomplish our goals, and avoids incurring the cost of manually maintaining this file which would be a pain for things like renames, or even just removing items from it that were fixed.

This is similar to how we check-in the generated api.md files: its not to prevent API changes, its to improve clarity about how changes impact something important (in that case API, in this case the topology of our dependency graph)